src/typegate/src/runtimes/substantial/worker.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+import { Context } from "./deno_context.ts";
+import { Err, Msg, Ok, WorkerData, WorkflowResult } from "./types.ts";
+
+let runCtx: Context | undefined;
+
+self.onmessage = async function (event) {
+  const { type, data } = event.data as WorkerData;
+  switch (type) {
+    case "START": {
+      const { modulePath, functionName, run, kwargs } = data;
+      const module = await import(modulePath);
+      // TODO: for python use the same strategy but instead call from native
+      const workflowFn = module[functionName];
+
```

Comment:
Idea being that for python, it would still in this worker file but instead we run a function from native that loads the artifact and calls the workflow `def`, the `Context` will be also written in python (or rust).